dkron/notifier.go

```diff
@@ -69,7 +69,10 @@ func (n *Notifier) report() string {
 }
 
 func (n *Notifier) buildTemplate(templ string, logger *logrus.Entry) *bytes.Buffer {
-	t := template.Must(template.New("report").Parse(templ))
+	t, e := template.New("report").Parse(templ)
+	if e != nil {
+		return bytes.NewBuffer([]byte("Failed to execute template: " + e.Error()))
```

Comment:
IMHO the error should be along the lines of 'Failed to parse template', to distinguish it from the other error below.

Reply:
You're right ! I change the message.